### ISSUE TITLE
Accuracy is very low for custom built model

### ISSUE THREAD

**Issue by techbala** [davidsandberg/facenet#766]
Ciao,
         I have trained the Inception Resnet V2 for lfw faces which is of size 321 by 321. Have tried the following steps,
         1. Align the images with 321 by 321
         2. Find the embeddings of these images based on the select_triplets and stored them as triples.txt and these embeddings were calculated from the InceptionResnetV2. There were 4.5 million pairs of triplets.
		 3. Trained InceptionResnetV2 with these triplets of 5000 epochs and saved the model
		 4. Using SVC with kernels linear, poly and rbf to test the images and all of them are giving only 42-45 images correctly out of 127 test images.
		 
	Source code copied in this page https://github.com/techbala/facenet.in.keras and would like to understand what went wrong in the above apporach.
	Thanks in advance.
	
Regards,
Bala
